src/mca/state/base/state_base_fns.c

```diff
@@ -761,7 +761,8 @@ void prte_state_base_track_procs(int fd, short argc, void *cbdata)
             parent.jobid = jdata->jobid;
             parent.vpid = PRTE_VPID_WILDCARD;
 
-            /* do not kill send the msg if ft prte is enabled */
+            /* if ft prte is enabled, a PMIx event has already been produced
+             * and this is redundant. */
             if (!prte_enable_ft) {
```

Comment:
prte_enable_ft is never true at the site of prun/mpiexec, even though the variable is registered and set (and is indeed set to true in prted)..